api/src/main/java/org/neo4j/ogm/config/DelegatingClassLoader.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (c) 2002-2020 "Neo4j,"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.neo4j.ogm.config;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Enumeration;
+import java.util.Iterator;
+import java.util.LinkedHashSet;
+import java.util.NoSuchElementException;
+import java.util.Set;
+
+/**
+ * This class loader delegates to a list of class loaders, which is by default a singleton list containing the class loader
+ * that has been used to load this class.
+ * <p>
+ * Before consulting the delegates, this the current threads class loader is used (if available). When neither that class loader
```

Comment:
```suggestion
 * Before consulting the delegates, the current threads class loader is used (if available). When neither the class loader
```
If I understand the comment correctly.